src/utils.ts

```diff
@@ -1,21 +1,58 @@
 'use strict'
 
-import { multihash, blake } from './deps.ts'
+import { base58btc, blake, colors } from './deps.ts'
+import { type Multibase } from './deps.ts'
 
 // TODO: implement a streaming hashing function for large files
-export function blake32Hash (data: string | Uint8Array): string {
+export function blake32Hash (data: string | Uint8Array): Multibase<'z'> {
   // TODO: for node/electron, switch to: https://github.com/ludios/node-blake2
+  // Note: Could also use `@multiformats/blake2`:
+  // import blake from "https://esm.sh/@multiformats/blake2"
+  // const uint8array = blake.blake2b.blake2b256.encode(data)
+  // Output is 32 byte long.
   const uint8array = blake.blake2b(data, undefined, 32)
-  // if you need Buffer: https://doc.deno.land/https://deno.land/std@0.141.0/io/mod.ts/~/Buffer
-  // const buf = Buffer.from(uint8array.buffer)
-  return multihash.toB58String(multihash.encode(uint8array, 'blake2b-32', 32))
+  // Previously we had to pass this 32 byte array to `multihash.encode()` to prepend
+  // four bytes of metadata before base-encoding to base58.
+  return base58btc.encode(uint8array)
```

Comment:
I'm pretty sure this is being used incorrectly, and because of that is generating the wrong prefix.

Here's proof:

```
-> % deno run --import-map=./vendor/import_map.json --allow-read=. --allow-write build/main.js hash build/main.js 
blake32Hash(build/main.js): zHnggvswR6hwn3qEpD7VXLzZWQXgu2JVuLFzdDEHj1mRV
-> % deno run --import-map=./vendor/import_map.json --allow-read=. --allow-write build/main.js hash deno.json 
blake32Hash(deno.json): zG5pBxvgCzEwrAJmYRAgWc2SgWB8AVSZqVYUTPz6Cmbta
```

Notice these two hashes have a different prefix.

If this was being correctly used they would have the same prefix.

Reply:
The prefix is just a 'z', hence the `Multibase<'z'>` return type for the` blake32Hash()` function. Is this incorrect?

Reply:
I'm not sure, but I thought the prefix had to be several bytes long? I can look into this more deeply...

Reply:
Hmm I think you're right, let me submit a change

Reply:
@snowteamer Here are two links that might be helpful:

- https://github.com/multiformats/js-multiformats#multihash-hashers
- https://github.com/multiformats/js-blake2

It's a little unclear to me though:

- Why they are doing different things
- What a "Block" is
- What a "CID" is

Can you investigate?